gulpfile.js

```diff
@@ -0,0 +1,66 @@
+var gulp = require("gulp");
```

Comment:
don't put building process file into github

Reply:
does it need a BFG Repo-Cleaner so it's not in the history?